src/crypto/utils/helpers.py

```diff
@@ -10,4 +14,16 @@ def service_credentials(service_acct: dict) -> Credentials:
 def parse_service_account_file(path: str) -> dict:
     with open(path, mode="r") as file:
         service_account_info = json.load(file)
-    return service_account_info
\ No newline at end of file
+    return service_account_info
+
+def service_account_absolute_path(service_account: str) -> str:
+    return f"{Path.cwd().parent.parent.parent}{config(service_account)}"
```

Comment:
have to go 3 up? just the way project structure is?

Reply:
yeah, not really a fan of this/might be a cleaner solution but it's a result of the project structure